DMTR-102.tex

```diff
@@ -44,183 +40,377 @@ \section{Introduction}
 \subsection{Objectives}
 \label{sect:objectives}
 
-Testing for the raw image from the DAQ to being written to the archive service at the LDF
-This test activity demonstrates the successful execution of the major components of the storing of raw image files from the DAQ to the DBB and then visualized through the LSST science platform (LSP).   This is at a very small scale and it to test the components of a proof of concept that they all work together.  
+Testing for the raw image from the DAQ to being written to the archive
+service at the LDF\\
+This test activity demonstrates the successful execution of the major
+components of the storing of raw image files from the DAQ to the DBB and
+then visualized through the LSST science platform (LSP). ~ This is at a
+very small scale and it to test the components of a proof of concept
```

Comment:
“it to test the components of a proof of concept that they all work together” isn't really English!

Reply:
took Leanne wording.   
done.